charts/gardener-extension-shoot-lakom-service/templates/deployment.yaml

```diff
@@ -69,7 +69,8 @@ spec:
           type: RuntimeDefault
       containers:
       - name: gardener-extension-{{ .Values.serviceName }}
-        image: {{ .Values.image }}
+        image: {{ include "image" . }}
```

Comment:
`image` template needs to be defined.

Reply:
It is already existing, isn't it?
https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/0adc6bb07ca55f5fe90278cf681063f0cd2553ef/charts/gardener-extension-shoot-lakom-service/templates/_helpers.tpl#L1-L7